app/controllers/application_controller.rb

```diff
@@ -11,6 +11,10 @@ def logged_in?
     !current_user.nil?
   end
 
+  def set_student
+    @student = CourseSessionParticipant.student.find_or_initialize_by(user: current_user, course_session_id: 2) #@course_session)
+  end
```

Comment:
need to unhardcode

Reply:
This is what Muffy is going to work on tomorrow. Should I unhardcode before the merge or leave as is?

Reply:
I'll take care of it.